crates/polars-core/src/series/series_trait.rs

```diff
@@ -327,6 +328,26 @@ pub trait SeriesTrait:
     /// Aggregate all chunks to a contiguous array of memory.
     fn rechunk(&self) -> Series;
 
+    fn rechunk_validity(&self) -> Option<Bitmap> {
```

Comment:
Copied from `impl<T> ChunkedArray<T>` so that it can be used without downcasting